b3sum/src/main.rs

```diff
@@ -177,8 +184,94 @@ fn read_key_from_stdin() -> Result<[u8; blake3::KEY_LEN]> {
     }
 }
 
+fn check_file(input: impl Read, mmap_disabled: bool) -> Result<u64> {
```

Comment:
It's a little confusing that you've got `check_files` and `check_file`, but it's the latter that loops over all the filenames?

Reply:
Correct.  I will rename things to make it clearer.